### PR TITLE
Yalla background

### PR DESCRIPTION
**Pull request by idonov8**

### CODE REVIEW

File: client/App/Sagas/StartupSaga.js [hunk 1]
```diff
@@ -20,8 +21,10 @@ export function* startup() {
   // Splash screen is shown till navigateAndReset is called ad end of this saga
   // ...
   yield put(PermissionsActions.permissionsRequest());
-
   // App moves to main screen on permissions granted.
+
+  // Make scans work in background
```
Comment (shacharbuda) [inline client/App/Sagas/StartupSaga.js:26]:
```suggestion
  // Make app work in background
```

File: client/App/App.js [hunk 2]
```diff
@@ -22,6 +25,11 @@ export default class App extends Component {
          */}
         <PersistGate loading={null} persistor={persistor}>
           <RootScreen />
+          <Button title="Stop working" onPress={() => {
```
Comment (shacharbuda) [inline client/App/App.js]: can be one liner
`onPress={() => stopForegroundService()}`
or even
`onPress={stopForegroundService}`

File: client/App/Services/AndroidForegroundService.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable prettier/prettier */
```
Comment (shacharbuda) [inline client/App/Services/AndroidForegroundService.js:1]: del

File: client/App/App.js (hunk 2 above)
Comment (idonov8) [inline client/App/App.js:28]: Not sure it's the right words decision

File: client/App/App.js [hunk 4]
```diff
@@ -1,10 +1,13 @@
+/* eslint-disable prettier/prettier */
```
Comment (shacharbuda) [inline client/App/App.js:1]: del

**Pull request merged**